### DISCUSSION TITLE
My Personal webpages not in master

### DISCUSSION THREAD

**Discussion by GiuseppeDiPalma** [community/community#23292] [Other Feature Feedback, Questions, & Ideas]
<p>Hi, everybody,<br>
I tried to create a repository to create my github page(for my account, not for my repositories), but the display name is: <a href="https://mynamesurname.github.io/mynamesurname.github.io" rel="nofollow noopener">https://mynamesurname.github.io/mynamesurname.github.io</a></p>
<p>I create a new repository public, and set name [<a href="http://mynamesurname.github.io" rel="nofollow noopener">mynamesurname.github.io</a>].</p>
<ul>
<li>Add README.md with &lt;h1&gt;HELLO&lt;/h1&gt;</li>
</ul>
<p>I go in SETTING and set GitHub Pages.</p>
<p>But,  <a href="https://mynamesurname.github.io/" rel="nofollow noopener">https://mynamesurname.github.io/</a> show only 404</p>
<p>while I would like it that way: <a href="https://mynamesurname.github.io/" rel="nofollow noopener">https://mynamesurname.github.io/</a></p>
<p>How come?</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/1X/fd5580c1d763627f4e43cbf135c8939d20a9b36d.png" data-download-href="/uploads/short-url/A95ZkJJUqxOi5q40rHrNEhvQJt3.png?dl=1" title="Annotazione 2020-04-22 173219.png" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/11930075/181076059-61f3f543-e531-4498-9e3a-182431dd7a47.png" alt="Annotazione 2020-04-22 173219.png" data-base62-sha1="A95ZkJJUqxOi5q40rHrNEhvQJt3" width="690" height="465" srcset="https://aws1.discourse-cdn.com/github/optimized/1X/fd5580c1d763627f4e43cbf135c8939d20a9b36d_2_690x465.png, https://aws1.discourse-cdn.com/github/original/1X/fd5580c1d763627f4e43cbf135c8939d20a9b36d.png 1.5x, https://aws1.discourse-cdn.com/github/original/1X/fd5580c1d763627f4e43cbf135c8939d20a9b36d.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/1X/fd5580c1d763627f4e43cbf135c8939d20a9b36d_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">Annotazione 2020-04-22 173219.png</span><span class="informations">950×641 30 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by lesleyanneb** [comment, marked as answer]
<p>Hi there,</p>
<p>Thank you for contributing to the GitHub community forum!</p>
<p>GitHub Pages has 2 different types of sites: User/Organization sites and Project sites:</p>
<p><a href="https://help.github.com/articles/user-organization-and-project-pages/" rel="nofollow noopener">https://help.github.com/articles/user-organization-and-project-pages/</a></p>
<p>User/Organization sites are created when you name the repository <code>username.github.io</code> and can only be served from the <code>master</code> branch.</p>
<p>Project sites can be created from any repository and served from the <code>master</code> branch, <code>gh-pages</code> branch, or the <code>/docs</code> folder inside the <code>master</code> branch. These sites are served from the <code>username.github.io/repository-name</code> URL.</p>
<p>If you’re encountering a 404 error it may be that the site is still building. It can take up to 10 minutes to finish building your site, and another 10 to propagate across our CDN, so you may have to wait after enabling GitHub Pages or pushing a new commit for your site to show the latest changes.</p>
<p>It is also worth noting that GitHub Pages serves an <code>index.html</code>, <code>index.md</code>, or <code>README.md</code> file from the root of your repository’s publishing source by default. If ones of these files are not present you might also encounter a 404 error. GitHub Pages is case sensitive so these files will need to be named <code>index.html</code>, <code>index.md</code>, or <code>README.md</code>. </p>
<p>I hope that helps as a start but please do let us know if you’re still running into issues.</p>
